packages/shared/ReactSymbols.js

```diff
@@ -39,6 +39,12 @@ export const REACT_ASYNC_MODE_TYPE = hasSymbol
 export const REACT_FORWARD_REF_TYPE = hasSymbol
   ? Symbol.for('react.forward_ref')
   : 0xead0;
+export const REACT_PLACEHOLDER_TYPE = hasSymbol
+  ? Symbol.for('react.placeholder')
+  : 0xead1;
+export const REACT_TIMEOUT_TYPE = hasSymbol
+  ? Symbol.for('react.timeout')
+  : 0xead2;
```

Comment:
These new symbols should be added to `ReactIs` and `getComponentName`.

Reply:
Probably also need a follow-up PR for DevTools to add the new type in so it doesn't show as "unknown"?

Reply:
Also test renderer 😄